server/pbench/bin/pbench-server-prep-version-001

```diff
@@ -0,0 +1,182 @@
+#! /bin/bash
+
+# set -x
+###########################################################################
+# load common things
+opts=$SHELLOPTS
+case $opts in
+    *xtrace*)
+        dir=$(dirname $(which $0))
+        PROG=$(basename $(which $0))
+        ;;
+    *)
+        dir=$(dirname $0)
+        PROG=$(basename $0)
+        ;;
+esac
+
+export CONFIG=/opt/pbench-server/lib/config/pbench-server.cfg
+
+. $dir/pbench-base.sh
+###########################################################################
+
+archive=$(getconf.py archive-dir pbench-server)
+quarantine=$(getconf.py quarantine-dir pbench-server)
+duplicates=$(getconf.py duplicates-dir pbench-server)
+
+errlog=$LOGSDIR/$PROG/$PROG.error
+if [[ "$archive" == "" ]] ;then
+    echo "Failed: \"getconf.py archive-dir pbench-server" >> $errlog
+    exit 1
+fi
+
+if [[ "$quarantine" == "" ]] ;then
+    echo "Failed: \"getconf.py quarantine-dir pbench-server" $errlog
+    exit 1
+fi
+
+if [[ "$duplicates" == "" ]] ;then
+    echo "Failed: \"getconf.py duplicates-dir pbench-server" $errlog
+    exit 1
+fi
+
+log_init $(basename $0) $LOGSDIR/$(basename $0)
+
+tmp=$TMP/$PROG.$$
+
+function quarantine () {
+    dest=$1
+    files="$@"
+
+    mkdir -o $dest || return $?
+    mv $files $dest || return $?
+}
+
+trap 'rm -rf $tmp' EXIT INT QUIT
+
+mkdir -p $tmp
+list=$tmp/list.check
+status=$tmp/status
+> $status
+results_dir=$(getconf.py receive-dir pbench-server)
+# check for results that are ready for processing
+find ${results_dir} -maxdepth 2 -name *.tar.xz.md5 > $list
+
+typeset -i ntbs=0
+typeset -i nerrs=0
+typeset -i nquarantined=0
+typeset -i ndups=0
+
+echo $TS
+while read ck ;do
+    # full pathname of tarball
+    tb=${ck%.md5}
+
+    # directory
+    tbdir=$(dirname ${tb})
+
+    # resultname: get the basename foo.tar.xz and then strip the .tar.xz
+    resultname=$(basename ${tb})
+    resultname=${resultname%.tar.xz}
+
+    # hostname is the last component of the directory part of the full path
+    controller=$(basename ${tbdir})
+
+    dest=$archive/$controller
+
+    # if there is a prefix file with the old name, normalize it now
```

Comment:
Given this is supporting new clients, why would we have a prefix file with an old name here?

Reply:
We would not - I got rid of this earlier, but I guess I never pushed the change.